pkg/endpoint/cache.go

```diff
@@ -187,3 +189,7 @@ func (ep *epInfoCache) RequireRouting() bool {
 func (ep *epInfoCache) RequireEndpointRoute() bool {
 	return ep.requireEndpointRoute
 }
+
+func (ep *epInfoCache) GetPolicyVerdictLogFilter() uint {
```

Comment:
One more question, if this becomes a uint32 when plumbed to the datapath then shouldn't the rest of the code in userspace assume this size as well? Just to make sure that we don't somehow end up truncating values at some point in the future.

Reply:
Yes, it is uint32 when plumbed to the datapath. Updated the patch to use uint32 for all places.